src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletController.java

```diff
@@ -14,4 +26,19 @@
 public class WalletController {
 
     private final WalletService walletService;
+
+    @PutMapping("/update")
+    public ResponseEntity<WalletDTO> updateWallet(UpdateWalletDTO updateWalletDto) {
+        WalletDTO walletDTO = walletService.updateWallet(updateWalletDto);
```

Comment:
`@RequestBody` and `@Valid` annotations missing

Reply:
Updated.

Reply:
Please remove the "/update". this the same: https://github.com/bycza-zagroda/track-expenses-app-backend/pull/53#discussion_r996501486